core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -291,59 +360,103 @@ public String toString() {
   public static final PropertyKey UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING =
       new Builder(Name.UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING)
           .setDefaultValue("")
+          .setDescription("Optionally, specify a preset s3 canonical id to Alluxio username "
+              + "static mapping, in the format \"id1=user1;id2=user2\". The AWS S3 canonical "
+              + "ID can be found at the console address "
+              + "https://console.aws.amazon.com/iam/home?#security_credential . Please expand "
+              + "the \"Account Identifiers\" tab and refer to \"Canonical User ID\".")
           .build();
   public static final PropertyKey UNDERFS_S3_PROXY_HOST =
       new Builder(Name.UNDERFS_S3_PROXY_HOST)
+          .setDescription("Optionally, specify a proxy host for communicating with S3.")
           .build();
   public static final PropertyKey UNDERFS_S3_PROXY_HTTPS_ONLY =
       new Builder(Name.UNDERFS_S3_PROXY_HTTPS_ONLY)
           .setDefaultValue(true)
+          .setDescription("If using a proxy to communicate with S3, determine whether to talk "
+              + "to the proxy using https.")
           .build();
   public static final PropertyKey UNDERFS_S3_PROXY_PORT =
       new Builder(Name.UNDERFS_S3_PROXY_PORT)
+          .setDescription("Optionally, specify a proxy port for communicating with S3.")
           .build();
   public static final PropertyKey UNDERFS_S3_THREADS_MAX =
       new Builder(Name.UNDERFS_S3_THREADS_MAX)
           .setDefaultValue(40)
+          .setDescription("The maximum number of threads to use for communicating with S3 and "
+              + "the maximum number of concurrent connections to S3. Includes both threads "
+              + "for data upload and metadata operations. This number should be at least as "
+              + "large as the max admin threads plus max upload threads. The default is 40 "
```

Comment:
fixed by removing "The default is 40 which is the sum of the default admin and upload thread pool sizes."